### DISCUSSION TITLE
Question for https://example.com/.well-known/privacy-sandbox-attestations.json

### DISCUSSION THREAD

**Discussion by hykim321** [privacysandbox/privacy-sandbox-dev-support#314] [General]
Hi, I have a question

https://developers.google.com/privacy-sandbox/relevance/enrollment?hl=ko

According to the documentation, my understanding is that the data must be sent using the root domain: https://example.com/.well-known/privacy-sandbox-attestations.json.

Can't we send this as a subdomain?
For example, I would like to send it to a subdomain in the form https://foo.example.com/.well-known/privacy-sandbox-attestations.json.
Please respond in this regard.

**Comment by GeorgiaFranklin** [comment]
Hello there,

Subdomain validation is not currently supported by our attestation validation process at this time. This is outlined in our [Enrollment Developer Guide ](https://developers.google.com/privacy-sandbox/relevance/enrollment?_gl=1*wiio6w*_up*MQ..*_ga*MTMyMjQxODg2Mi4xNzIwODExMTYz*_ga_JPRHSQDH0G*MTcyMDgxMTE2Mi4xLjAuMTcyMDgxMTE2Mi4wLjAuMA..) See section titled "Upload your Attestation file". 

Can you provide more explanation as to why you would prefer to host the file on a sub-domain?

**Reply by jyfhsszq** [reply]
Most of the companies usually don't host service for https://example.com. All traffics to https://example.com are redirected to https://www.example.com by default. To support attestation file under https://example.com, it requires big exceptional rule change on infra. It is huge risk for security and stability. Several same concerns were reported as well in this ticket. Could you please change your policy to make subdomain available? https://github.com/privacysandbox/attestation/issues/36
